rules/still-taking-manual-meeting-notes/rule.md

```diff
@@ -0,0 +1,68 @@
+---
+type: rule
+tips: ""
+title: Do you still take manual meeting notes?
+seoDescription: Stop wasting time and mental energy on manual meeting notes, use
+  AI-powered assistants like Read AI to automate your transcripts, summaries,
+  and follow-ups.
+uri: still-taking-manual-meeting-notes
+authors:
+  - title: Louis Roa
+    url: https://www.ssw.com.au/people/louis-roa/
+    img: https://www.ssw.com.au/people/static/Louis-Roa-Sketch-43386bcbae2db3e94bdb99b5bdae4f96.jpg
```

Comment:
Hi @louisroa8189 

When using you SSW People URL, you don't need the image

- [ ] 1. Please fix